### ISSUE TITLE
Typing error its transform and not transfrom

### ISSUE THREAD

**Issue by akucharczyk** [YIZHUANG/react-multi-carousel#299]
For example additionalTransfrom

**Comment by caleyg** [comment]
Was this resolved? I still see the typo in `2.8.2`

```
declare class Carousel extends React.Component<CarouselProps, CarouselInternalState> {
    static defaultProps: {
        slidesToSlide: number;
        infinite: boolean;
        draggable: boolean;
        swipeable: boolean;
        arrows: boolean;
        renderArrowsWhenDisabled: boolean;
        containerClass: string;
        sliderClass: string;
        itemClass: string;
        keyBoardControl: boolean;
        autoPlaySpeed: number;
        showDots: boolean;
        renderDotsOutside: boolean;
        renderButtonGroupOutside: boolean;
        minimumTouchDrag: number;
        className: string;
        dotListClass: string;
        focusOnSelect: boolean;
        centerMode: boolean;
        additionalTransfrom: number; 👈 
        pauseOnHover: boolean;
        shouldResetAutoplay: boolean;
        rewind: boolean;
        rtl: boolean;
        rewindWithAnimation: boolean;
    };

```

**Comment by caleyg** [comment]
Was this resolved? I still see the typo in `2.8.2` [here](https://github.com/YIZHUANG/react-multi-carousel/blob/master/src/Carousel.tsx#L54). Could this be reopened?

```
declare class Carousel extends React.Component<CarouselProps, CarouselInternalState> {
    static defaultProps: {
        slidesToSlide: number;
        infinite: boolean;
        draggable: boolean;
        swipeable: boolean;
        arrows: boolean;
        renderArrowsWhenDisabled: boolean;
        containerClass: string;
        sliderClass: string;
        itemClass: string;
        keyBoardControl: boolean;
        autoPlaySpeed: number;
        showDots: boolean;
        renderDotsOutside: boolean;
        renderButtonGroupOutside: boolean;
        minimumTouchDrag: number;
        className: string;
        dotListClass: string;
        focusOnSelect: boolean;
        centerMode: boolean;
        additionalTransfrom: number; 👈 
        pauseOnHover: boolean;
        shouldResetAutoplay: boolean;
        rewind: boolean;
        rtl: boolean;
        rewindWithAnimation: boolean;
    };

```
